templates/content/node--oe-project--full.html.twig

```diff
@@ -0,0 +1,63 @@
+{#
+/**
+ * @file
+ * Project full display.
+ */
+#}
+{% if not content.oe_subject.isEmpty() %}
+  {% set _badges = [] %}
+  {% for _badge in content.oe_subject|field_value %}
+    {% set _badges = _badges|merge([{
+      label: _badge|raw,
+      rounded_pill: true,
+    }]) %}
+  {% endfor %}
+{% endif %}
+{{ pattern('content_banner', {
+  background: 'gray',
+  title: label,
+  content: content.oe_teaser,
+  image: image,
+  attributes: create_attribute().addClass(['ps-0']),
+  badges: _badges,
+}) }}
+{% macro field_with_header(field) %}
+  {% if field|field_value is not empty %}
+    <h3 class="fw-bold mb-4 pt-3" id="{{ field['#inpage_nav_id'] }}">{{ field|field_label }}</h3>
+    {{ field }}
+  {% endif %}
+{% endmacro %}
+{% set _project_details %}
+  {{ content.group_project_details }}
```

Comment:
The group title still shows up even if the group is empty.
But I don't have a quick solution. So let's merge ignore it for now and do a follow up.